awx/main/management/commands/cleanup_jobs.py

```
@@ -49,6 +49,41 @@ def dt_to_partition_name(tbl_name, dt):
+    if not job_pks:
+        return
+
+    with connection.cursor() as cursor:
+        for i in range(0, len(job_pks), JHS_CHUNK_SIZE):
+            chunk = job_pks[i : i + JHS_CHUNK_SIZE]
+            placeholders = ','.join(['%s'] * len(chunk))
+
+            cursor.execute(
+                f"UPDATE main_host SET last_job_host_summary_id = NULL"
+                f" WHERE last_job_host_summary_id IN"
+                f" (SELECT id FROM main_jobhostsummary WHERE job_id IN ({placeholders}))",
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)